v2/dtos/requests/event.go

```diff
@@ -9,7 +9,7 @@ import (
 	"encoding/json"
 
 	"github.com/edgexfoundry/go-mod-core-contracts/errors"
-	"github.com/edgexfoundry/go-mod-core-contracts/v2"
+	contractsV2 "github.com/edgexfoundry/go-mod-core-contracts/v2"
```

Comment:
don't use alias unless the name conflict

Reply:
Alias removed.